plugin/neoterm.vim

```diff
@@ -28,7 +28,11 @@ endfunction
 let g:neoterm_statusline = ""
 
 if !exists("g:neoterm_shell")
-  let g:neoterm_shell = &shellcmd . ' ' . substitute(&shellcmdflag, '[-/]c', '', '')
+  if exists('&shellcmdflag')
```

Comment:
afaict, the fundamental error is: `E113: Unknown option: shellcmd`, and thus either branch will still fail.

I believe (uncertain) that the 2 branches below both need `s/\<shellcmd\>/shell/`.
This snippet WFM:

```vim
  if !exists("g:neoterm_shell")
    if exists('&shellcmdflag')
      let g:neoterm_shell = &shell . ' ' . substitute(&shellcmdflag, '[-/]c', '', '')
    else
      let g:neoterm_shell = &shell
    end
  end
```